SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -182,3 +182,8 @@ uint32 USpatialGDKSettings::GetRPCRingBufferSize(ERPCType RPCType) const
 
 	return DefaultRPCRingBufferSize;
 }
+
+bool USpatialGDKSettings::UseRPCRingBuffer() const
+{
+	return bUseRPCRingBuffers || bEnableUnrealLoadBalancer;
```

Comment:
Would be helpful to add a comment explaining why this is necessary.

Reply:
Can we disable the `UseRPCRingBuffers` option when the load balancer is enabled? Might help save trolling people accidentally

Reply:
You mean graying out the boolean in the settings ?

Reply:
+1 to that. I think greying it out but not forcing it to be ticked. Although the intention is eventually to get rid of the option to turn it off so the fine details don't matter so much.